cmd/zsysd/completion.go

```diff
@@ -11,18 +11,52 @@ import (
 func installCompletionCmd(rootCmd *cobra.Command) {
 	prog := rootCmd.Name()
 	var completionCmd = &cobra.Command{
-		Use:   "completion",
+		Use:   "completion [bash|zsh|powershell]",
```

Comment:
Here, you agree (see below comment) that the syntax should have an optional argument :)